packages/mdc-tab/README.md

```diff
@@ -43,9 +43,106 @@ npm install --save @material/tab
 ```html
 <button class="mdc-tab" role="tab" aria-selected="false">
   <div class="mdc-tab__content">
-    <span class="mdc-tab__icon">heart</div>
     <span class="mdc-tab__text-label">Favorites</div>
   </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+#### Text Label and Icon
+
+You can use an icon in place of or with a text label.
+
+##### Text Label
+
```

Comment:
This is just the same example as the first HTML code snippet. I think this is redundant